libr/core/Makefile

```diff
@@ -11,6 +11,7 @@ OBJS+=fortune.o hack.o vasm.o patch.o cbin.o corelog.o rtr.o cmd_api.o
 OBJS+=carg.o canal.o project.o gdiff.o casm.o disasm.o cplugin.o
 OBJS+=vmenus.o vmenus_graph.o vmenus_zigns.o zdiff.o citem.o
 OBJS+=task.o panels.o pseudo.o vmarks.o anal_tp.o anal_objc.o blaze.o cundo.o
+OBJS+=core_annotated_code.o
```

Comment:
Rename this as `cannotated_code`, similar to how it's done with other files (e.g. `cio`, `cconfig`, `cfile`, `cundo`, etc.)

Reply:
Done.